tests/rt.conf

```diff
@@ -19,6 +19,10 @@ RUN     | cpld_restart_bmark_p8
 COMPILE | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8                                                             | - wcoss_cray  wcoss2.intel              | fv3 |
 RUN     | cpld_debug_p8                                                                                                           | - wcoss_cray  wcoss2.intel              | fv3 |
 
+# Waves and aerosol off for computing fluxes in mediator
+COMPILE | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_sfcocn -DCMEPS_AOFLUX=ON                                                 | - wcoss_cray                            | fv3 |
+RUN     | cpld_control_noaero_p8_agrid                                                                                        | - wcoss_cray                            | fv3 |
```

Comment:
I think previous cpld_control_noaero_p8 does not have wave, but it was updated to include wave in the latest develop branch. Maybe we can consider changing the test name or enabling this exchange grid test to include wave and aerosol later.

Reply:
@junwang-noaa I did not know it. It could be nice to update it to be consistent with the existing configurations.  We could rename the test but that will require creating baselines again or at least changing folder names. Let me know what do you think?  If you want to rename the test, do you have any name in your mind. Thanks.